litmus-portal/graphql-server/server.go

```diff
@@ -80,10 +89,8 @@ func main() {
 	mongoClient := mongodb.Initialize(client)
 
 	var mongodbOperator mongodb.MongoOperator = mongodb.NewMongoOperations(mongoClient)
-	// TODO: remove this when all packages shift to interface pattern
-	mongodb.Operator = mongodbOperator
```

Comment:
There are some global variables still there in the database package I believe we need to remove those too right?

Reply:
Move some variables to local in this commit. If any variables needed to be fixed, Please tell me! [fix: move variables global to local](https://github.com/litmuschaos/litmus/pull/3937/commits/ab3b0567e7e9af616849fc92181ff347d5bf1fe9)